presubmit_tests/single-instance-on-gcp.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+gcs_bucket="gs://oracle-toolkit-presubmit-artifacts"
+# Append BUILD_ID to the file name to ensure each zip file gets a unique name.
+# This prevents one test from deleting the file while it's still in use by another concurrently running test.
+# For available Prow-injected environment variables, see:
+# https://docs.prow.k8s.io/docs/jobs/#job-environment-variables
+toolkit_zip_file_name="oracle-toolkit-${BUILD_ID}.zip"
+tfvars_file="./presubmit_tests/single-instance.tfvars"
+instance_name="github-presubmit-si-${BUILD_ID}"
+deployment_name="presubmit-si-${BUILD_ID}"
+location="us-central1"
+project="gcp-oracle-benchmarks"
```

Comment:
nit: Our gcloud commands seem to rely on the default project, so perhaps we should retrieve it via gcloud config get-value project
 rather than hard-coding here.